app/models/concerns/queue_it/queable.rb

```diff
@@ -31,32 +35,32 @@ def get_next_in_queue
       end
     end
 
-    def formatted_queue(node_attribute)
+    def formatted_queue(nodable_attribute)
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Metrics/AbcSize: Assignment Branch Condition size for formatted_queue is too high. [<3, 26, 5> 26.65/25]